src/pybind/mgr/nfs/export.py

```diff
@@ -74,6 +74,15 @@ def write_obj(self, conf_block: str, obj: str, config_obj: str = '') -> None:
             ExportMgr._check_rados_notify(ioctx, config_obj)
             log.debug("Added %s url to %s", obj, config_obj)
 
+    def read_obj(self, obj: str) -> Optional[str]:
+        with self.mgr.rados.open_ioctx(self.pool) as ioctx:
+            ioctx.set_namespace(self.namespace)
+            try:
+                return ioctx.read(obj, 1048576).decode()
+            except Exception as e:
+                log.debug(f"Failed to read {obj}: {e}")
```

Comment:
We probably don't want to handle exceptions here. If we do it then we will have to check the return values in `get_nfs_cluster_config()`. Instead these exceptions can be handled directly in `get_nfs_cluster_config()`

Reply:
changed thist o catch ObjectNotFound only (which is what i was going for)